cli/running/running.go

```diff
@@ -659,6 +659,50 @@ func Stop(run *InstanceCtx) error {
 	return nil
 }
 
+// Kill the Instance.
+func Kill(run *InstanceCtx) error {
+	pid, err := process_utils.KillProcess(run.PIDFile)
+	if err != nil {
+		return err
+	}
+
+	// remove PID files, because due to
+	// SIGKILL watchdog can't cleanup itself
+	cleanup(run)
+
+	// tarantool 1.10 does not have a trigger on terminate a process.
+	// So the socket will be closed automatically on termination and
+	// we need to delete the file.
+	if _, err := os.Stat(run.ConsoleSocket); err == nil {
+		os.Remove(run.ConsoleSocket)
+	}
+
+	fullInstanceName := GetAppInstanceName(*run)
+	log.Infof("The Instance %s (PID = %v) has been killed.", fullInstanceName, pid)
+
+	return nil
+}
+
+// Quit the Instance.
+func Quit(run *InstanceCtx) error {
```

Comment:
We are not going to modify this var, lets pass it by value.

Reply:
changed